docs/source/guide/zne-5-theory.myst

```diff
@@ -0,0 +1,110 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+
+# What is the theory behind ZNE?
+
+
+Zero noise extrapolation (ZNE) {cite}`Giurgica_Tiron_2020_arXiv` is an error
+mitigation technique used to extrapolate the noiseless expectation value of an
+observable from a range of expectation values computed at different noise levels.
+In Mitiq, this process works in two steps:
+
+- **Step 1: Intentionally scale noise** by manually inserting gates in a manner
+that keeps the logic of input circuit unchanged. The number of gates inserted is
+dependent on the scale factor. Different scale factor values will calculate the
+expectation value at different noise leveles.
+
+- **Step 2: Extrapolate to the noiseless limit** by fitting a curve [via one of
+the various methods available in Mitiq](https://mitiq.readthedocs.io/en/stable/apidoc.html#module-mitiq.zne.inference)
+to the expectation values calculated at different noise levels to extrapolate the
+noiseless expectation value.
```

Comment:
```suggestion
- **Step 2: Extrapolate to the noiseless limit**. This can be done by
fitting a curve (often called *extrapolation model*) to the expectation values measured at different noise levels
to extrapolate the noiseless expectation value.

Better to keep the theory section Mitiq-independent.

Reply:
Previously I made a mistake and wrote _Better to keep the theory section Mitiq-independent_. in the code snipped.
This is now updated.